google/cloud/grpc_options_test.cc

```diff
@@ -107,6 +107,8 @@ TEST(GrpcChannelArguments, MakeChannelArguments) {
   expected.SetString("foo", "bar");
   expected.SetString("baz", "quux");
   expected.SetUserAgentPrefix("user_agent");
+  expected.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS, 86400000);
+  expected.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, 60000);
```

Comment:
A few comments here....

I think we should move these into a new `TEST(GrpcChannelArguments, MakeChannelArgumentsDefaults) {}`. It's just good to have focused tests. That way if one of the tests fail, we will know whether it is the defaulting logic or the combining logic.

I think the defaults test should verify that...
1. if a user does not set the values, our defaults are used.
2. if a user does set the values, their values are used. (not our defaults).

For part 1, I think we do not need to be checking for equality of the values. That is, our default keepalive time is 24 hours. Would we care if it was actually 23 hours? or 25 hours? I don't think so. I think it is sufficient to check that *any* value is set. Alternatively, we could check that the value is at least 1 hour or something.
```cc
EXPECT_GT(value, std::chrono::milliseconds(std::chrono::hours(1)).count());
```